src/groundStation/system.py

```diff
@@ -3007,5 +3023,58 @@ def __init__(self):
                         }
                     },
                 }
+            },
+            'DFGM': {
+                'port': 19,
+                'subservice': {
+                    'DFGM_RUN': {
+                        'subPort': 0,
+                        'inoutInfo': {
+                            'args': ['>u4'],
+                            'returns': {
+                                'err': '>b'
+                            }
+                        }
+                    },
+                    'DFGM_START': {
+                        'subPort': 1,
+                        'inoutInfo': {
+                            'args': None,
+                            'returns': {
+                                'err': '>b'
+                            }
+                        }
+                    },
+                    'DFGM_STOP': {
+                        'subPort': 2,
+                        'inoutInfo': {
+                            'args': None,
+                            'returns': {
+                                'err': '>b'
+                            }
+                        }
+                    },
+                    'DFGM_GET_HK': {
+                        'subPort': 3,
+                        'inoutInfo': {
+                            'args': None,
+                            'returns': {
+                                'err': '>b',
+                                'Core Voltage': '>f',
+                                'Sensor Temperature': '>f',
```

Comment:
Are these not stored as integers on the satellite?

Reply:
Yes you are correct. They should be >u2 for uint16_t instead of >f, which I'll fix sometime tomorrow!